mopro-ffi/src/circom/mod.rs

```diff
@@ -175,22 +166,99 @@ pub fn generate_circom_proof_wtns(
 pub fn verify_circom_proof(
     proof_lib: ProofLib,
     zkey_path: String,
-    proof: Vec<u8>,
-    public_inputs: Vec<u8>,
+    proof_result: CircomProofResult,
```

Comment:
I think we can change all the `proof_ret` to `proof_result`. It will be more clear to me

Reply:
fixed in [a0551c9](https://github.com/zkmopro/mopro/pull/372/commits/a0551c983b38309cd18d6e21617883a74b541b1f)